packages/allure-cypress/src/utils.ts

```diff
@@ -5,6 +5,13 @@ import { ALLURE_REPORT_STEP_COMMAND, ALLURE_REPORT_SYSTEM_HOOK } from "./model.j
 import type { CypressCommand, CypressHook, CypressSuite, CypressTest } from "./model.js";
 import { getAllureTestPlan } from "./state.js";
 
+export const defaultRuntimeConfig = {
```

Comment:
Let's name the constant using `UPPER_SNAKE_CASE`

Reply:
Agree!